src/monodroid/jni/monodroid-glue.cc

```diff
@@ -1192,6 +1192,12 @@ MonodroidRuntime::init_android_runtime (
 		native_to_managed_index = internal_timing->start_event (TimingEventKind::NativeToManagedTransition);
 	}
 
+	if (mappingXml != nullptr && mappingXmlLen > 0) {
+		init.mappingXml             = env->GetByteArrayElements (mappingXml, nullptr);
+		init.mappingXmlLen          = mappingXmlLen;
+	}
+	log_warn (LOG_DEFAULT, "# jonp: mappingXml? len=%i, xml=%p", init.mappingXmlLen, init.mappingXml);
```

Comment:
Ditto logging

Reply:
For now, i'll leave this in so that there is *some* runtime indicator that mapping is being used, but i'll move it into the above `if` block so it's only printed *when used*.